.github/actions/step_sdk_setup/action.yml

```diff
@@ -28,6 +28,17 @@ runs:
       echo "########### pip freeze ###########"
       pip freeze
     working-directory: ${{ inputs.scriptPath }}
+  - uses: snok/install-poetry@v1
+  - name: 'Build and install: promptflow-tracing'
+    shell: pwsh
+    run: |
+      Set-PSDebug -Trace 2
+      poetry build
```

Comment:
Just poetry install will do the whole work.

Reply:
Refined.